python/feathub/metric_stores/metric_store_config.py

```diff
@@ -0,0 +1,65 @@
+#  Copyright 2022 The FeatHub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from datetime import timedelta
+from enum import Enum
+from typing import Any, Dict, List
+
+from feathub.common.config import BaseConfig, ConfigDef
+from feathub.common.validators import in_list, gt
+
+
+class MetricStoreType(Enum):
+    PROMETHEUS = "prometheus"
+
+
+METRIC_STORE_PREFIX = "metric_store."
+
+METRIC_STORE_TYPE_CONFIG = METRIC_STORE_PREFIX + "type"
+METRIC_STORE_TYPE_DOC = "The type of the metric store to use."
+
+METRIC_STORE_REPORT_INTERVAL_CONFIG = METRIC_STORE_PREFIX + "report_interval"
+METRIC_STORE_REPORT_INTERVAL_DOC = "The interval to report metrics."
+
+METRIC_STORE_NAMESPACE_CONFIG = METRIC_STORE_PREFIX + "namespace"
+METRIC_STORE_NAMESPACE_DOC = "The namespace of the metric store."
```

Comment:
Should we document the impact of this config? For example, how does this config affects FeatHub output?